templates/docker-rootful.yaml

```diff
@@ -68,3 +68,7 @@ message: |
   docker context use lima-{{.Name}}
   docker run hello-world
   ------
+portMonitors:
+  docker:
+    sockets:
+    - "/var/run/docker.sock"
```

Comment:
You don't really need to put quotes around paths that don't contain template variables. But I guess it doesn't hurt either.